docs/data/material/customization/css-theme-variables/default-tokens-viewer.md

```diff
@@ -0,0 +1,63 @@
+# Default tokens viewer
+
+<p class="description">A preview of the default CSS theme variables for built-in light and dark color schemes</p>
+
+The following tokens are generated when using `CssVarsProvider` in your application.
+
+## Colors
+
+The color tokens are generated from `colorSchemes`:
+
+{{"component": "modules/material-ui/PaletteTokensViewer.js"}}
+
+## Font
+
+The font tokens are generated from `typography`:
+
+{{"component": "modules/material-ui/FontTokensViewer.js"}}
+
+:::success
+Using font variable is an alternative way to `theme.typography` when you cannot access the theme in Javascript.
+The font variable should be used with the CSS `font` property like this:
+
+```css
+.text {
+  font: var(--mui-font-h1);
+}
+```
+
+:::
+
+## Spacing
+
+The variable `--mui-spacing` is generated from the `spacing` field with a default of `8px`.
+
+Below are the usage examples:
+
+<codeblock>
+
+```js JS
+theme.spacing(1, 2); // var(--mui-spacing) calc(2 * var(--mui-spacing))
+```
+
+```css CSS
+.some-class {
+  margin: calc(var(--mui-spacing) * 2);
+}
+```
```

Comment:
I'm not sure this example is clear. Maybe users expect both snippets to be equivalent, and the first one using the multiple arity version of the function might cause confusion. I'd keep the example to the bare minimum.